File: generators/app/templates/middle-tier/src/main/scala/spring-boot/Application.scala
package <%= packageName %>

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.Import
import org.springframework.web.bind.annotation.GetMapping

import <%= packageName %>.config.SpaWebAppConfiguration

@SpringBootApplication
class <%= mainClassName %>

object Main {

    def main(args: Array[String]): Unit = SpringApplication.run(classOf[<%= mainClassName %>])

}

File: generators/app/templates/middle-tier/src/main/java/spring-boot/Application.java
package <%= packageName %>;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class <%= mainClassName %> {

    public static void main(String[] args) {
        SpringApplication.run(<%= mainClassName %>.class, args);
    }

}
